Formula/geode.rb

```diff
@@ -0,0 +1,36 @@
+class Geode < Formula
+  desc "In-memory Data Grid for fast transactional data processing"
+  homepage "https://geode.apache.org"
+  url "http://apache.org/dyn/closer.cgi?filename=incubator/geode/1.0.0-incubating.M2/apache-geode-1.0.0-incubating.M2.tar.gz&action=download"
+  version "1.0.0-incubating.M2"
+  sha256 "8447912c6d893008dd03917722d01abff4853861ce3f34c2bc95b60c51675cea"
+
+  bottle :unneeded
+  depends_on :java
```

Comment:
Nit but please leave an empty line between the `bottle` modifier and the first `depends_on`, Thanks!